pkgs/applications/science/math/getdp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, cmake, gfortran, blas, liblapack, openmpi }:
+
+stdenv.mkDerivation rec {
+  name = "getdp-${version}";
+  version = "3.0.3";
+  src = fetchurl {
+    url = "http://getdp.info/src/getdp-${version}-source.tgz";
+    sha256 = "1ng96vrbcpb6f757mv9sz4pww7h6bhlwwkzxjapgmrjw2avq081z";
+  };
+
+  buildInputs = [ cmake gfortran blas liblapack openmpi ];
+
+  enableParallelBuilding = true;
```

Comment:
I think it's default behavior of Cmake builds. In this case, this is unnecessary.

Reply:
Removed the `enableParallelBuilding = true;`, builds still with multiple cores. Thanks for the feedback!